src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,32 @@ function createAllowPolicy(
   };
 }
 
+function getHeaderValue(
+  headers: Record<string, string> | undefined,
+  name: string
+): string | undefined {
+  if (!headers) {
+    return undefined;
+  }
+  const lcName = name.toLowerCase();
+  for (const h of Object.keys(headers)) {
```

Comment:
Please write `headerName` instead of `h`

Reply:
Fixed.

You should consider adding a contribution or style guide to the repository to avoid requesting those kind of changes. (Or does the repo already have such a file and I missed it?)

Reply:
No, we don't, but if you look at the surrounding code, it should be clear that variables are semantically named. I think it's generally good style to use descriptive names. Minification is what the minifiers do. ;)